src/julia.h

```diff
@@ -625,7 +625,6 @@ typedef struct _jl_module_t {
     arraylist_t usings;  // modules with all bindings potentially imported
     jl_uuid_t build_id;
     jl_uuid_t uuid;
-    size_t primary_world;
```

Comment:
Is this no longer relevant? I was using it to execute code in a frozen world.

Reply:
This used to be an id to find which module added a particular method, but we don't need that anymore since we have a better version of that now